packages/relay/src/lib/errors/JsonRpcError.ts

```diff
@@ -137,4 +140,10 @@ export const predefined = {
     code: -32007,
     message: 'Historical balance data is available only after 15 minutes.'
   }),
+  'CONTRACT_REVERT': (errorMessage?: string) => new JsonRpcError({
+    name: 'Contract revert executed',
+    code: 3,
```

Comment:
Where are we setting this code off of as it seems to break current convention

Reply:
I tested the same use case with the Alchemy's rpc relay and received this code. It's also the same code mentioned in the linked issue.